app/controllers/users.js

```diff
@@ -0,0 +1,11 @@
+const { signUp } = require('../services/users');
+
+exports.signUp = (req, res, next) => {
+  const { firstName, lastName, email, password } = req.body;
+  signUp(firstName, lastName, email, password)
+    .then(user => {
+      res.status(201).send(`A new User was created!
+        firstName: '${user.firstName}', lastName: '${user.lastName}', email: '${user.email}'`);
```

Comment:
theres no point in giving a response here unless its the user info in a frontend friendly format like a json but the initial message has no practical use